src/style.css

```diff
@@ -255,6 +255,41 @@ textarea {
   align-items: center;
 }
 
+@media only screen and (max-width: 600px) {
```

Comment:
this is also not the correct media query, because mobile view doesn't start at 600px but 650px